.circleci/continue_config.yml

```diff
@@ -363,17 +363,25 @@ jobs:
           destination: sql-artifacts
 
   compile-integration-sql-java-linux:
-    working_directory: ~/openlineage/integration/sql/iface-java
-    docker:
-      - image: cimg/openjdk:8.0
-    resource_class: medium
+    parameters:
+      image:
+        type: string
+      resource_class:
+        type: string
+    working_directory: ~/openlineage/integration/sql
+    machine:
+      image: ubuntu-2004:current
+    resource_class: << parameters.resource_class >>
     steps:
       - *checkout_project_root
-      - run: bash script/compile.sh
+      - run: docker run -it -v $PWD:/code << parameters.image >> bash -c 'cd /code/iface-java && bash script/compile.sh'
       - persist_to_workspace:
-          root: ../
+          root: ./
           paths:
-            - target/debug/libopenlineage_sql_java.so
+            - target/debug/*.so
+      - store_artifacts:
```

Comment:
For some reason this step is getting following output:
```
Uploading /home/circleci/openlineage/integration/sql/iface-java/target/debug to sql-artifacts
  No artifact files found at /home/circleci/openlineage/integration/sql/iface-java/target/debug
Total size uploaded: 0 B
```

Reply:
@JDarDagran could you recheck?

Reply:
@JDarDagran removed intermediate step artifacts. They were never needed.